controller/src/bin/xcvradm.rs

```diff
@@ -632,6 +836,109 @@ fn load_known_write_data<R: Read>(rdr: &mut R, kind: InputKind) -> anyhow::Resul
     }
 }
 
+#[derive(Clone, ValueEnum, PartialEq)]
+enum StatusFields {
+    /// The port number of the switch.
+    Port,
+    /// The power state of the transceiver module.
+    Status,
+}
+
+#[derive(Clone, ValueEnum, PartialEq)]
+enum PowerFields {
+    /// The port number of the switch.
+    Port,
+    /// The power state of the transceiver module.
+    Power,
+    /// Flag to indicate a software override of the modules power state.
+    Override,
+}
+
+#[derive(Clone, ValueEnum, PartialEq)]
+enum IdentifyFields {
+    /// The port number of the switch.
+    Port,
+    /// SFF-8024 identifier of the transceiver module.
+    Ident,
+    /// ??? - Needs details
```

Comment:
This is a text description of the module's management interface. These come from the spec SFF-8024. For example, a lot of our 100G modules have an identifier of `0x11`, which that spec lists as "QSFP28 or later with SFF-8636 management interface". That's what we print here.